common-tasks/backup-restore.md

```diff
@@ -22,31 +22,32 @@ As of Qubes R2B3, these functions are integrated into the Qubes VM Manager GUI.
 Creating a Backup (R4.0 and later)
 -----------------
 
-1. In **Qubes VM Manager**, click **System** on the menu bar, then click **Backup Qubes** in the drop-down list. This brings up the **Qubes Backup VMs** window.
+1. Go to **Applications menu -> System Tools -> Backup Qubes**. This brings up the **Qubes Backup VMs** window.
 
 2. Move the VMs that you want to back up to the right-hand **Selected** column. VMs in the left-hand **Available** column will not be backed up.
 
-   **Note:** A VM must be shut down in order to be backed up. Currently running VMs appear in red.
-
    You may choose whether to compress backups by checking or unchecking the **Compress the backup** box. Normally this should be left on unless you have a specific reason otherwise.
    
    Once you have selected all desired VMs, click **Next**.
 
 3. Select the destination for the backup:
 
    If you wish to send your backup to a (currently running) VM, select the VM in the drop-down box next to **Target AppVM**.
-   If you wish to send your backup to a [USB mass storage device](/doc/stick-mounting/), first mount the device in a VM, then select the mount point inside that VM as the backup destination.
+   If you wish to send your backup to a [USB mass storage device](/doc/stick-mounting/), you can use the directory selection widget to mount a connected device (under "Other locations" item on the left); or first mount the device in a VM, then select the mount point inside that VM as the backup destination.
```

Comment:
The new URL is `/doc/usb/`. This old one will get redirected there, but we might as well spare the redirect while we're editing the page anyway.